css/index.css

```diff
@@ -212,7 +204,6 @@ footer {
         justify-content: center;
         flex-flow: column;
         align-items: center;
-        border: 1px solid purple;
         margin: 0 0 0 0;
```

Comment:
This can be shortened to:
`  margin: 0, 0;` (set margin top bottom, margin left right to 0)
or
`  margin: 0;` (set all margins to 0)